### ISSUE TITLE
403 Forbidden on WebUI

### ISSUE THREAD

**Issue by thisispiggy** [nextcloud/nextcloudpi#282]
### What version of NextCloudPi are you using? ( eg: v0.17.2 )
v0.32.2

### What is the base image that you installed on the SD card? ( eg: NextCloudPi_07-21-17 )

NextCloudPi_10-29-17

### Expected behavior

Accessing nextcloudpi.local:4443 prompting username and password

### Actual behaviour

Accessing 4443 prompts 403 error:

"Forbidden
You don't have permission to access / on this server."

### Steps to reproduce, from a freshly installed image

Basically a fresh image plus wifi enabled.

### Include logs

admin logs: no details here
apache logs: I couldn't copy the logs. But it says: 

access check of 'localhost' to /favicon.ico failed, reason: unable to get the remote host name, referer: https://nextcloudpi.local:4443:/
access check of 'localhost' to / failed, reason: unable to get the remote host name
client denied by server configuration /var/www/ncp-web/

**Comment by Pant** [comment]
Hi @thisispiggy,
Thanks for reporting this.

Please try to add a 's', like httpS... 
Example: https://nextcloudpi.local:4443
